flash/core/utilities/imports.py

```diff
@@ -20,6 +20,8 @@
 from typing import Callable, List, Tuple, Union
 
 import pkg_resources
+from lightning_utilities.core.imports import compare_version as _compare_version
+from lightning_utilities.core.imports import module_available as _module_available
```

Comment:
Let's keep the names `compare_version` and `module_available`. :) 
```suggestion
from lightning_utilities.core.imports import compare_version
from lightning_utilities.core.imports import module_available
```